### PR TITLE
Repo config updates

### PR DESCRIPTION
**Pull request by lelia**
Repo config files pulled from #9 

# Changes

* Updated `CODEOWNERS` and `stale.yml`
* Added Dependabot config (`dependabot.yml`)
* Added YAML Lint config (`.yamllint.yml`)

### CODE REVIEW

**Review by justaugustus — APPROVED** [review]
🎉🎉🎉🎉🎉🎉🎉🎉

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
 # ref: https://github.com/probot/settings#security-implications
+
+# Maintainers
+# - Stephen Augustus (@justaugustus)
+# - Jason DeTiberus (@detiber)
+# - Márk Sági-Kazár (@sagikazarmark)
+# - Ed Warnicke (@edwarnicke)
+# - Shannon McFarland (@shmcfarl)
+# - Lelia Bray-Musso (@lelia)
+# - Sam Betts (@sambetts-cisco)
```
Comment (justaugustus) [inline .github/CODEOWNERS:11]: Plus @svrnm

**Review by justaugustus — CHANGES_REQUESTED** [review]
@lelia -- Ope, let's add Severin and alpha-sort the list on last names before merging!

File: .github/CODEOWNERS (hunk 1 above)
Comment (lelia) [reply .github/CODEOWNERS:11]: O right, he wasn't yet an org member when I filed this :D

**Review by justaugustus — APPROVED** [review]
Thanks @lelia!

**Pull request merged**